diff --git a/CMakeLists.txt b/CMakeLists.txt
index 17f3dce..64e0a2d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -222,7 +222,7 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
         ERROR_QUIET
         OUTPUT_STRIP_TRAILING_WHITESPACE
     )
-    CHECK_INCLUDE_FILES(proc/readproc.h READPROC_H_FOUND)
+    check_include_file_cxx (proc/readproc.h READPROC_H_FOUND)
     if (READPROC_H_FOUND)
         add_definitions (-DHAVE_READPROC_H)
         find_library (proc_LIBRARY NAMES proc procps)
